### ISSUE TITLE
New top-level XML tag causing complaints

### ISSUE THREAD

**Issue by oddhack** [KhronosGroup/Vulkan-Hpp#1950]
FYI, in (edit: the 1.3.294 spec update) we added a new 'feature' tag inside 'require' / 'remove' blocks which is causing some complaints from the Hpp generator.

**Comment by asuessenbach** [comment]
Thanks for that hint, I'll add support for those new tags. Shouldn't be too hard.

I have a question, though: those new <feature>s have a "name" and a "struct". For most of them, that "name" is a member of that "struct". But not all. Is that correct?
Those who behave differently are:
```
                <feature name="bufferDeviceAddress" struct="VkPhysicalDeviceDescriptorIndexingFeatures"/>
```
and
```
                <feature name="shaderSampledImageArrayDynamicIndexing" struct="VkPhysicalDeviceDescriptorIndexingFeaturesEXT"/>
                <feature name="shaderStorageBufferArrayDynamicIndexing" struct="VkPhysicalDeviceDescriptorIndexingFeaturesEXT"/>
```

**Comment by oddhack** [comment]
Best to ask @Tobski who came up with the new tags.

**Comment by asuessenbach** [comment]
... and, besides that: what does those new tags tell me?
Does it mean, that those features are required to be supported by those extensions/versions?

**Comment by oddhack** [comment]
> ... and, besides that: what does those new tags tell me? Does it mean, that those features are required to be supported by those extensions/versions?

That's what the schema documentation is for, see if this helps: https://registry.khronos.org/vulkan/specs/1.3/registry.html#tag-required-feature

**Comment by asuessenbach** [comment]
Thanks for that link!
It answers my questions and tells me that the inconsistencies I've identified above actually are errors in vk.xml.

**Comment by asuessenbach** [comment]
Thanks for that link!
It answers my questions and makes clear that the inconsistencies I've identified actually are errors in the vk.xml.

**Comment by asuessenbach** [comment]
Resolved by #1953.
